docs/activemq.md

```diff
@@ -0,0 +1,75 @@
+# `activemq` Metrics Receiver
+
+The `activemq` metrics receiver will be collected through the [Otel JMX Metric Gatherer](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/jmx-metrics).
+.
+
+## Prerequisites
+
+JMX support must be enabled in the [broker configuration](https://activemq.apache.org/jmx).
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   |--------------------| ---         |
+| `endpoint`            | `localhost:1099`   | Must be in the form of `host:port`.|
+| `username`            | not set by default | The configured username if JMX is configured to require authentication. |
+| `password`            | not set by default | The configured password if JMX is configured to require authentication. |
+| `collection_interval` | `60s`              | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    activemq:
+      type: activemq
+      endpoint: localhost:1099
+      collection_interval: 30s
```

Comment:
Remove the `collection_interval` here so that we can provide the bare minimum config for users in the sample config?